### DISCUSSION TITLE
Working Linux Version?

### DISCUSSION THREAD

**Discussion by adamwparker** [bnpr/Malt#116] [Q&A]
I'm running 2.93.1 and Malt Development Version is not working for me. If anyone has it working for them, could you please tell me your Blender version? Or am I just overlooking some simple solution - I added a simple PixelOutput following Miguel's tutorial, I don't even get a black material - just grey. Thanks!

**Comment by pragma37** [comment]
This might be related to #101.

Try https://github.com/bnpr/Malt/releases/tag/debug-shader-compilation-latest.

If that doesn't work, please, open a bug report with your log file.
https://github.com/bnpr/Malt#bug-reports
